pinot/connection.go

```diff
@@ -84,8 +84,8 @@ func formatArg(value interface{}) (string, error) {
 		hexString := fmt.Sprintf("%x", v)
 		return fmt.Sprintf("'%s'", hexString), nil
 	case time.Time:
-		// For pinot type - TIMESTAMP - convert to ISO8601 format and enclose in single quotes
```

Comment:
will this introduce backward incompatible issue say if left side column to compare is a string ?

Reply:
A valid point if timestamp values are stored in a string column. However if that is done then shouldn't the client also pass timestamp argument as formatted string? But writing timestamp arg as ISO8601 format string will work generally in all cases. I was trying to push this down to driver so that indexes are used without doing anything special and deliver good performance to any client using this driver but anyways.

I think then it will need to be handled at client level by using `CAST(? AS TIMESTAMP)` and passing time value as argument to the driver.

Reply:
That's fair.
I think to be safe, I will publish a new version and state this as a backward incompatible changes then :)

Reply:
https://github.com/startreedata/pinot-client-go/pull/48